ADO service principal note (#6796)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/git/setup-azure.md

sidebar_label: "Set up Azure DevOps"
---

# Set up Azure DevOps <Lifecycle status="enterprise"/>

:::note

> The service principal integration for Azure DevOps is currently available for select Enterprise accounts. dbt Labs will be implementing a phased rollout in the near future across all Enterprise accounts. If you'd like the integration enabled now, please [contact us](https://www.getdbt.com/contact/) or your account manager.

Author: ```suggestion
The service principal integration for Azure DevOps is currently available for select Enterprise accounts. dbt Labs will be implementing a phased rollout shortly across all Enterprise accounts. 
```
